_posts/2016-06-28-position_sheffield.md

```diff
@@ -1,6 +1,6 @@
 ---
 layout: post
-title: Full time position, University of Sheffield, United Kingdom
+title: Full time position has expired, University of Sheffield, United Kingdom
 ---
 
 [A full time research software engineer position](http://www.jobs.ac.uk/job/ANT812/research-software-engineer/) starting June/July 2016 for two years.
```

Comment:
If you'd like, instead of changing the title above, you can just add here that the **job advert has expired** (or just close this PR, since it's now obvious after more than half a year - sorry for not getting back earlier).